docs.yaml

```diff
@@ -35,9 +35,15 @@ links:
     href:  irc://irc.freenode.net/datastax-drivers
   - title: Releases
     href:  https://github.com/datastax/php-driver/releases
+rewrites:
+  - http://downloads.datastax.com/php-driver/1.0.0/: http://downloads.datastax.com/php-driver/
+  - http://downloads.datastax.com/php-driver/1.1.0/: http://downloads.datastax.com/php-driver/
+  - http://downloads.datastax.com/php-driver/1.2.0/: http://downloads.datastax.com/php-driver/
+  - http://datastax.github.io/php-driver/api/Cassandra/interface.Type/: https://docs.datastax.com/en/developer/php-driver/latest/api/Cassandra/class.Type/
+  - http://datastax.github.io/php-driver/features/http://localhost:8000/features/datatypes/: https://docs.datastax.com/en/developer/php-driver/latest/features/
 versions:
   - name: 1.3
-    ref: v1.3.1
+    ref: HEAD
```

Comment:
Are any of these changes required? It seems to me only the `features/README.md` is the only thing that needed to be fixed,